src/traveler.js

```diff
@@ -48,6 +47,25 @@ class Traveler {
     })
     this.currentTrip = currentTrip;
   }
+
+  calculateTravelBudget(destinations) {
+    let year = moment('2020-01-01');
+    let yearTrips = this.allTrips.filter(trip => {
+      return moment(new Date(trip.date)).isAfter(year);
+    })
```

Comment:
I think for the scope of this project, 2020 is the correct year. If you wanted to make it more dynamic, you could find today = moment(Date.now()) and then use moments .add() or .subtract() method to filter all trips between today and a year ago from today. If you want, I'd recommend looking into the .add() and .subtract() for this.